### DISCUSSION TITLE
Whether appwrite have super admin level access?

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [appwrite/appwrite#1493] [Q&A]
In appwrite instance super admin level access will help in following means
1. To manage all user on instance level and also o kick out user entirely or suspend account.
2. To manage and view all projects created as well as user wise projects can be accessed
3. to understand project wise analysis.

I faced a situation that project created by a user, he accidently leave the project now he is not accessible and how to get access back to that particular project without such super admin level access?

**Comment by eldadfux** [comment]
This feature is definitely on the Appwrite backlog. We would love to get more community feedback and other ideas that can go well with this "super admin" mode

**Comment by MercusChan** [comment]
I expect this is also support through cli
